wings/market/coinbase_pro_market.pyx

```diff
@@ -0,0 +1,816 @@
+import aiohttp
+import asyncio
+from async_timeout import timeout
+import hmac, hashlib, base64
+from requests.auth import AuthBase
+from decimal import (
+    Decimal
+)
+import json
+import logging
+import time
+from typing import (
+    Dict,
+    List,
+    Optional,
+)
+from web3 import Web3
+
+from wings.clock cimport Clock
+from wings.events import (
+    TradeType,
+    MarketEvent,
+    BuyOrderCompletedEvent,
+    SellOrderCompletedEvent,
+    OrderFilledEvent,
+    OrderCancelledEvent,
+    BuyOrderCreatedEvent,
+    SellOrderCreatedEvent,
+    MarketReceivedAssetEvent,
+    MarketWithdrawAssetEvent,
+    MarketTransactionFailureEvent)
+from wings.market.market_base import (
+    MarketBase,
+    OrderType,
+)
+from wings.order_book_tracker import (
+    OrderBookTrackerDataSourceType
+)
+from wings.order_book cimport OrderBook
+from wings.tracker.coinbase_pro_order_book_tracker import CoinbaseProOrderBookTracker
+from wings.cancellation_result import CancellationResult
+from wings.transaction_tracker import TransactionTracker
+from wings.wallet.wallet_base import WalletBase
+from wings.wallet.wallet_base cimport WalletBase
+
+s_logger = None
+s_decimal_0 = Decimal(0)
+
+
+class CoinbaseProAuth(AuthBase):
+    def __init__(self, api_key, secret_key, passphrase):
+        self.api_key = api_key
+        self.secret_key = secret_key
+        self.passphrase = passphrase
+
+    def __call__(self, request):
+        body = "" if request.body is None else request.body.decode("utf8")
+        request.headers.update(self.get_headers(request.method, request.path_url, body))
+        return request
+
+    def get_headers(self, method: str, path_url: str, body: str = "") -> Dict[str, any]:
```

Comment:
use uppercase "Any" from typing for any type

```from typing import Any```

Reply:
done